Begun footer.

pwaller815 opened the pull request.

Commit:
Begun footer.

Commit:
Resolved conflicts.

Commit:
Resolved conflicts p2.

pwaller815 committed:
Merge branch 'main' into pwaller

EthanCloin committed:
Bookings Feature:

- Update schema of Bookings
- Add generation of Bookings records to database init function

EthanCloin commented on app/templates/navbar.html:
```diff
@@ -1,15 +1,10 @@
 <header class="navbar">
-  <img
-    class="logo"
-    src="{{ url_for('static', filename='/images/NextStep-logo.webp') }}"
```
do you mean to switch back from the url_for function to string filepath? would recommend keeping the function in place, tho it's most handy when referencing a blueprint path.

EthanCloin commented on app/templates/footer.html:
```diff
@@ -0,0 +1,58 @@
+                        </svg>
+                        Phone:
+                    </p>
+                    <a class="footer-p footer-util" href="tel:+1234567890">+1 (904) 664-NEXT(6398)</a>
```
href here doesn't line up with the business's number

EthanCloin commented on app/static/css/footer.css:
```diff
@@ -0,0 +1,89 @@
+.footer {
+    margin-top: 100px;
+    position: relative;
+    background-image: url(../images/Teaching.webp);
```
the image and color overlay here looks great

EthanCloin reviewed (commented):
overall looks good! take a look at the href and switch string urls to use the url_for function where possible.

EthanCloin committed:
checkpoint: calendar broken

Commit:
Merge branch 'main' into pwaller

Commit:
Updated to comments.

Commit:
Prettier formatting applied.

EthanCloin committed:
Bookings Page:

- add form for booking submission
- add button to popup form (hardcoded booking_id)
- move calendar into detail page

Commit:
Added Poppins font.

Commit:
Merge branch 'ecloin' into pwaller

Commit:
Added tutor info with styles to booking page.

Commit:
Began styling time slots.
